api/endpoints.go

```diff
@@ -113,6 +113,8 @@ func post(i *jsonAPIHandler, path string, w http.ResponseWriter, r *http.Request
 		i.POSTPost(w, r)
 	case strings.HasPrefix(path, "/ob/bulkupdatecurrency"):
 		i.POSTBulkUpdateCurrency(w, r)
+	case strings.HasPrefix(path, "/ob/sendordermessage"):
+		i.POSTSendOrderMessage(w, r)
```

Comment:
The semantics around this endpoint should be clear that this is for resending. Maybe `/ob/resendordermessage` and `POSTResendOrderMessage` instead?